src/jit/flowgraph.cpp

```diff
@@ -21871,7 +21886,8 @@ void Compiler::fgInline()
             expr = stmt->gtStmtExpr;
 
             // See if we can expand the inline candidate
-            if ((expr->gtOper == GT_CALL) && ((expr->gtFlags & GTF_CALL_INLINE_CANDIDATE) != 0))
+            if (expr->IsCall() && (((expr->gtFlags & GTF_CALL_INLINE_CANDIDATE) != 0) ||
```

Comment:
I think it will be more readable as
```
if (expr->IsCall())
{
	 GenTreeCall* call = expr->AsCall();
	 if (expr->IsGuardedDevirtualizationCandidate() || call.IsGuardedDevirtualizationCandidate())
```

Reply:
Sure, will update.